app/src/main/java/org/xbmc/kore/ui/sections/settings/SettingsFragment.java

```diff
@@ -217,4 +233,76 @@ public boolean onPreferenceClick(Preference preference) {
             }
         });
     }
+
+    private void setupLanguagePreference(final ListPreference languagePref) {
+        Locale[] locales = getLocales(BuildConfig.SUPPORTED_LOCALES);
+
+        Arrays.sort(locales, new Comparator<Locale>() {
+            @Override
+            public int compare(Locale o1, Locale o2) {
+                return o1.getDisplayName().compareToIgnoreCase(o2.getDisplayName());
```

Comment:
Should be `getDisplayName(currentLocale);`

Reply:
fixed

Reply:
Ok